code/public/index.html

```diff
@@ -13,7 +13,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
```

Comment:
You might want to clear out the commented out bits of this file.

Reply:
Thanks! fixed, I always neglect the index.html these days 😅